src/integrationTest/java/io/aiven/kafka/connect/jdbc/ConnectRunner.java

```diff
@@ -75,11 +77,22 @@ void start() {
         final Plugins plugins = new Plugins(workerProps);
         final StandaloneConfig config = new StandaloneConfig(workerProps);
 
+        final ConnectorClientConfigOverridePolicy connectorClientConfigOverridePolicy =
+                new AllConnectorClientConfigOverridePolicy();
```

Comment:
Required by https://github.com/apache/kafka/pull/6624 ([KIP-458](https://cwiki.apache.org/confluence/display/KAFKA/KIP-458%3A+Connector+Client+Config+Override+Policy)), which landed in 2.3.0.